templates/index.rst

```diff
@@ -0,0 +1,29 @@
+.. LSST DM Stack  documentation master file, created by
+   sphinx-quickstart on Tue May 12 10:44:33 2015.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+
+LSST DM Stack documentation
+============================
+
+Contents:
+
+.. toctree::
+   :maxdepth: 1
+
+   template.forTasks
```

Comment:
If we make the title of the task template realistic, you can give it a sensible title here as:

```
Task Template <template.forTasks>
```